Tests/Marketplace/marketplace_services.py

```diff
@@ -1564,36 +1493,48 @@ def assert_upload_bucket_version_matches_release_notes_version(self,
             f'current branch version: {latest_release_notes}\n' \
             'Please Merge from master and rebuild'
 
-    def get_rn_files_names(self, modified_rn_files_paths):
+    def get_rn_files_names(self, diff_files_list):
         """
-
         Args:
             modified_rn_files_paths: a list containing all modified files in the current pack, generated
             by comparing the old and the new commit hash.
         Returns:
             The names of the modified release notes files out of the given list only,
             as in the names of the files that are under ReleaseNotes directory in the format of 'filename.md'.
-
         """
         modified_rn_files = []
-        for file_path in modified_rn_files_paths:
-            modified_file_path_parts = os.path.normpath(file_path).split(os.sep)
-            if self.RELEASE_NOTES in modified_file_path_parts:
-                modified_rn_files.append(modified_file_path_parts[-1])
+        for file_path in diff_files_list:
+            file_a_path = file_path.a_path
+            if not self.is_pack_release_notes_file(file_a_path):
+                continue
+            logging.debug(f"Found file path '{file_a_path}' as a modified release notes file of pack '{self._pack_name}'")
+            modified_file_path_parts = os.path.normpath(file_a_path).split(os.sep)
+            modified_rn_files.append(modified_file_path_parts[-1])
```

Comment:
is there a reason not to do:
```suggestion
            modified_file_path_parts = os.path.normpath(file_a_path).split(os.sep)
            modified_rn_files.append(Path(file_a_path).stem)
```

Reply:
`Path(file_a_path).stem` returns the name without the suffix (`.md`)
Maybe it can be okay if it's handled in other places but I prefer to leave it that way for now